src/seedu/addressbook/commands/SetTagCommand.java

```diff
@@ -0,0 +1,55 @@
+package seedu.addressbook.commands;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.tag.Tag;
+
+
+/**
+ * Sets the tags of the person identified using the last displayed index.
+ * Modifies and displays the edited person
+ */
+public class SetTagCommand extends Command {
+
+    public static final String COMMAND_WORD = "settags";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sets the tags of the person "
+            + "identified by the index number in the last shown person listing.\n"
+            + "Parameters: INDEX [t/TAG]\n"
+            + "Example: " + COMMAND_WORD + " 1 t/friend t/evil";
+
+    static final String MESSAGE_VIEW_NEW_TAGS = "New Tags: %1$s";
+
+    private Set<Tag> tags;
```

Comment:
Does this means that it can add multi-tags?

Reply:
Good idea!